_includes/articles/dong-btse-operation.md

```diff
@@ -0,0 +1,11 @@
+BTSE uses segwit, BIP32 HD wallets, and multisig key management to reduce their operational burdens and improve fund safety. For this Optech field report, we interviewed BTSE staff to learn how their exchange operations have benefited from these Bitcoin technologies.
```

Comment:
I suggest linking this first `BTSE` to their website.  I think that's appropriate promotion and it's what we did for Xapo in the previous report: https://bitcoinops.org/en/xapo-utxo-consolidation/

Reply:
I'll link to their Twitter! https://twitter.com/btsecom